scripts/convert-svgs-to-data-uri.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs');
+const path = require('path');
+const SVGO = require('svgo');
+
+const svgDir = './scripts/svgs/';
```

Comment:
It would be nice if the script would accept the path as an argument.

`node svgs-to-data.js ./svgs`

Sure, for convenience we could hardcode some path in package.json script, but it would be good to be able to run in on any folder.

Reply:
Also, I'm not sure about `scripts/svgs` - if we mean scripts to be a folder for scripts, it wouldn't expect to put icons there. I think some folder in root project dir would be better.

Other (less convienient, but safer approach) would be not to hardcode any path and require user to pass it. This way we avoid hardcoding the path, adding it to ignore, etc.

We should of course document the usage of the script. Seems like a good candidate to start HACKING.md file.